src/realmGeol.ttl

```diff
@@ -166,18 +166,23 @@ soreag:GeologicFeature rdf:type owl:Class ;
                      rdfs:subClassOf [ rdf:type owl:Restriction ;
                                        owl:onProperty sorel:hasRealm ;
                                        owl:allValuesFrom sorea:Geosphere
-                                     ] ;
-                     dcterms:contributor [
-                        a sdo:Organization ;
-                        sdo:name "Geological Survey of Queensland" ;
-                        sdo:identifier <http://linked.data.gov.au/org/gsq> ;
-                     ] ;                                     
+                                     ] ;                                    
                      skos:altLabel "Geologic Feature" , "Geological Feature" ;
-                     skos:definition "A geologic feature is a conceptual feature a that is hypothesized to exist coherently in the Earth that results from geological processes"@en ;
```

Comment:
Tricky to handle in a computable ontology - metaphysics and things in the conceptual (let alone hypothesised conceptual) realm can have great difficulty mixing when logical coherency needs to be preserved.